src/cmds/cloud/deleteThing.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-console */
+import yargs from 'yargs';
+import chalk from 'chalk';
+import { Client } from '@cesarbr/knot-cloud-sdk-js';
+import options from './utils/options';
+import getFileCredentials from './utils/getFileCredentials';
+
+const deleteThing = async (args) => {
+  const client = new Client({
+    hostname: args.server,
+    port: args.port,
+    token: args.token,
+    protocol: args.protocol,
+  });
+  await client.connect();
+  await client.unregister(args.id);
+  await client.close();
+};
+
+yargs
+  .config('credentials-file', path => getFileCredentials(path))
+  .command({
+    command: 'delete-thing <id>',
+    desc: 'Delete thing <id>',
+    builder: (_yargs) => {
+      _yargs
+        .options(options)
+        .positional('id', {
+          describe: 'Thing ID',
+        });
+    },
+    handler: async (args) => {
+      try {
+        await deleteThing(args);
+        console.log(chalk.green('Thing sucessfully deleted!'));
```

Comment:
Should this be logged? 
I think even without a feedback the intuition is that command executed successfully (similar to Linux tools behavior).

Reply:
Agreed.